Assets/Scripts/Gamestate/PlayerManager.cs

```diff
@@ -284,7 +285,7 @@ void OnDestroy()
         }
         if (TryGetComponent(out PlayerMovement playerMovement))
         {
-            playerMovement.onMove -= UpdateHudOnMove;
+            playerMovement.OnMove -= UpdateHudOnMove;
```

Comment:
Please remember to unsubscribe from the UpdateHudOnJump delegate as well ❗

Reply:
Fixed